src/main/java/uk/gov/ida/saml/metadata/EidasMetadataConfigurationImpl.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.ida.saml.metadata;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.client.JerseyClientConfiguration;
+import uk.gov.ida.verifyserviceprovider.configuration.HubEnvironment;
+
+import java.net.URI;
+import java.security.KeyStore;
+import java.util.Optional;
+
+public class EidasMetadataConfigurationImpl extends EidasMetadataConfiguration {
```

Comment:
Can we rename this to something more helpful. For example, `EnvironmentBackedMetadataConfiguration`

Reply:
This should go away with the merge of `EuropeanIdentityConfiguration` and `EnvironmentBackedMetadataConfiguration`